build-tools/create-packs/Directory.Build.props

```diff
@@ -3,6 +3,7 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
+    <PackageType>DotnetPlatform</PackageType>
```

Comment:
/cc @rolfbjarne at some point, you might add this to the iOS/Mac packages. Just so people can't attempt to add a `<PackageReference/>` to them.